classes/core/Application.php

```diff
@@ -245,6 +245,14 @@ public function hasCustomizableReviewerRecommendation(): bool
         return true;
     }
 
+    /**
+     * Define if the application has continuous publication functionality
+     */
+    public function hasContinuousPublication(): bool
```

Comment:
I don't think this function is necessary here and in pkp-lib -- I don't see it used anywhere in the PRs.